mne/viz/backends/renderer.py

```diff
@@ -121,11 +121,7 @@ def get_3d_backend():
         The 3d backend currently in use. If no backend is found,
         returns ``None``.
     """
-    try:
-        backend = _get_3d_backend()
-    except RuntimeError:
-        return None
-    return backend
+    return MNE_3D_BACKEND
```

Comment:
It looks like this will no longer cause any checking to actually be done. And indeed it appears to be the case -- on `master` this seems correct:
```
$ python -c "import mne; print(mne.viz.get_3d_backend())"
mayavi
$ MNE_3D_BACKEND=pyvista python -c "import mne; print(mne.viz.get_3d_backend())"
pyvista
```
On this PR it does not:
```
$ python -c "import mne; print(mne.viz.get_3d_backend())"
None
$ MNE_3D_BACKEND=pyvista python -c "import mne; print(mne.viz.get_3d_backend())"
None
```

Reply:
Oh. I always just set the backend via `viz.set_3d_backend()`. Need to rethink this entire PR then.

Reply:
I reverted that particular change, could you tell me if it works for you now as expected?

Reply:
Yes it works again